README.md

```diff
@@ -142,3 +156,32 @@ different files for each version of `mongodb` from `v1.0.0` through to the lates
   ]
 }
 ```
+
+## Testing
+The module includes a suite of unit and functional tests which should be used to
+verify that your changes don't break existing functionality.
+
+All tests are stored in `tests/` and are written using
+[Node-Tap](https://www.npmjs.com/package/tap) with the extension `.tap.js`.
+
+To run the full suite, run: `npm test`.
+
+Individual test scripts include:
+
+```
+npm run lint
+npm run unit
+```
+
+## Support
+New Relic hosts and moderates an online forum where you can interact with New Relic employees as well as other customers to get help and share best practices. Like all official New Relic open source projects, there's a related Community topic in the New Relic Explorers Hub. You can find this project's topic/threads here: https://discuss.newrelic.com/c/support-products-agents/node-js-agent/.
```

Comment:
Not convinced we actually want to be pointing folks to the same explorer's hub for this test utilities library. Or that we really need one at all.